guides/plugins/plugins/administration/using-the-data-grid-component.md

```diff
@@ -0,0 +1,58 @@
+# Using the data grid component
+
+The data grid component makes it easy to render tables with data. It also supports hiding columns or scrolling horizontally when many columns are present. This guide shows you how to use it.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance and full access to both the files, as well as the command line and preferably registered module.
+Of course you'll have to understand JavaScript, but that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Creating a Template for the data grid component
+
+Let's create the simplest Template we need in order to use the [`sw-data-grid`](https://component-library.shopware.com/components/sw-data-grid).
+
+{% code title="<plugin root>/src/Resources/app/administration/src/module/swag-example/swag-example.html.twig" %}
+```html
+<div>
+    <sw-data-grid :dataSource="dataSource" :columns="columns">
+	</sw-data-grid>
+</div>
+```
+
+This template will be used in a new component. Learn how to override existing components [here](./customizing-components.md) 
+
+## Declaring the Data
+
+Since this is a very basic example the following code will just statically assign data to the `dataSource` and `columns` data attribute. 
+If load data and render that instead please consult the guide [`how to use the data handling`](./using-the-data-handling.md)
+
+{% code title="<plugin root>/src/Resources/app/administration/src/module/swag-example/index.js" %}
```

Comment:
Path says "module", the code says `Component.register".

Reply:
i canged it to a component path